snuba/util.py

```diff
@@ -238,24 +239,28 @@ def is_condition(cond_or_list):
     )
 
 
-def all_referenced_columns(query_body):
+def all_referenced_columns(query: Query):
```

Comment:
I would like to move this as a public method of Query, but I cannot do that since it depends on a function that is in util and I cannot make Query depend on util (circular).
Taking a step back, the method this depends on is `columns_in_expr`. This is likely to become a method owned by an Expression class soon, so the problem will disappear